src/modules/personenkontext/persistence/dbiam-personenkontext.repo.ts

```diff
@@ -276,4 +276,39 @@ export class DBiamPersonenkontextRepo {
     public async isRolleAlreadyAssigned(id: RolleID): Promise<boolean> {
         return (await this.findByRolle(id)).length > 0;
     }
+
+    public async getPersonenKontexteWithExceedingBefristung(): Promise<Record<string, Personenkontext<true>[]>> {
+        const filters: QBFilterQuery<PersonenkontextEntity> = {
+            personId: {
+                $in: this.em
+                    .createQueryBuilder(PersonenkontextEntity, 'pk')
+                    .select('pk.person_id')
+                    .where({ befristung: { $lt: new Date() } })
+                    .getKnexQuery(),
+            },
+        };
```

Comment:
Not a fan of the hardcoded strings for the `person_id`

Shouldn't a MikroORM query like this work?
"Get me every Personenkontext, that belongs to a Person, who has at least one expired personenkontext"

```typescript
personId: {
    personenKontexte: {
        $some: {
            befristung: {
                $lt: new Date()
            }
        }
    }
}
```

Reply:
works, I like it more as well! Thanks